core/commands/storage/path.go

```diff
@@ -203,15 +203,15 @@ var PathCapacityCmd = &cmds.Command{
 			return err
 		}
 		return cmds.EmitOnce(res, &PathCapacity{
-			FreeSpace: usage.Free,
+			FreeSpace: humanize.Bytes(usage.Free),
```

Comment:
Do not print as humanize, it will break frontend api.

You can add a new field called `HumanizedFreeSpace` to display this string while keeping the original bytes.

Reply:
updated